neurons/validator.py

```diff
@@ -572,10 +572,11 @@ async def run_step(self):
             random.randint(1, pt.dataset.SubsetFalconLoader.max_pages)
             for _ in range(self.config.pages_per_eval)
         ]
+        # Use the largest sequence length and truncate as necessary for smaller ones.
         batches = list(
             pt.dataset.SubsetFalconLoader(
                 batch_size=constants.batch_size,
-                sequence_length=constants.sequence_length,
+                sequence_length=constants.SEQUENCE_LENGTH_2,
```

Comment:
Note this will half the number of batches in general.

Reply:
Should we choose a larger number of batches / batch size for the 7B model? It seems more efficient to run less frequently but with larger batches since model loading takes a notable amount of time

Reply:
Updated to double the batches to 6 to keep this steady.